deep_agent/aegra/graph.py

```
@@ -115,6 +152,30 @@ async def _ensure_startup() -> None:  # noqa: E402
     _startup_done = True
 
 
+def _resolve_personalization_uid(user: Any, token: str | None) -> str | None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `user`

(ANN401)